docs/account.md

```diff
@@ -171,3 +173,37 @@ const txHistory = account.getTransactionHistory();
 ```
 account.disconnect();
 ```
+
+### Events 
+
+```json
+const {EVENTS} = require('@dashevo/wallet-lib');
+const {READY} = EVENTS;
+const doSomethingWhenReady = (info) => {...}
+account.events.on(READY, doSomethingWhenReady);
+
+```
+
+Events types : 
```

Comment:
Would be cool to put it in a table
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables

Reply:
Done :)